src/foam/comics/v2/DAOCreateView.js

```diff
@@ -78,6 +79,16 @@ foam.CLASS({
       }
     }
   ],
+  messages: [
+    {
+      name: 'SUCCESS_REQUEST_MESSAGE',
+      message: 'An approval request has been created.'
```

Comment:
1. Why something so specific in a generic class? 
2. We can't have reference to Approvals in foam as they are on the nanopay side at the moment.

Reply:
I agree.